docker/konduit-python/start-server.py

```diff
@@ -71,37 +71,39 @@ def schema_type(python_type):
         "pythonOutputs": dict(outputs)
     }
 
-    pythonPipelineStep = PythonPipelineStep(input_names=["default"],
-                                            input_column_names={"default": [key for key, _ in inputs]},
-                                            input_schemas={"default": [schema_type(value) for _, value in inputs]},
-                                            output_names=["default"],
-                                            output_column_names={"default": [key for key, _ in outputs]},
-                                            output_schemas={"default": [schema_type(value) for _, value in outputs]},
-                                            python_configs={"default": pythonConfig})
+    default = "default"
+
+    pythonPipelineStep = PythonPipelineStep(input_names=[default],
+                                            input_column_names={default: [key for key, _ in inputs]},
+                                            input_schemas={default: [schema_type(value) for _, value in inputs]},
+                                            output_names=[default],
+                                            output_column_names={default: [key for key, _ in outputs]},
+                                            output_schemas={default: [schema_type(value) for _, value in outputs]},
+                                            python_configs={default: pythonConfig})
 
     inference = InferenceConfiguration(serving_config=serving_config,
                                        pipeline_steps=[pythonPipelineStep])
 
     server = Server(config=inference,
-                    extra_start_args='-Xmx8g',
+                    extra_start_args='-agentlib:jdwp=transport=dt_socket,server=y,suspend=n,address=5005',
                     jar_path='konduit.jar')
 
     server.start()
+    sleep_time = 20
+    print('Process started. Sleeping ' + str(sleep_time) + ' seconds.')
+    time.sleep(sleep_time)
 
-    client = Client(input_names=[key for key, _ in inputs],
-                    output_names=[key for key, _ in outputs],
+    client = Client(input_names=["default:x", "default:y"],
+                    output_names=[default],
                     input_type='NUMPY',
                     endpoint_output_type='NUMPY',
-                    url='http://localhost:' + str(port))
+                    url='http://localhost:' + str(port), timeout=2000000)
 
     data_input = {
-        'x': np.ones([2, 2]),
-        'y': np.ones([2, 2])
+        'default:x': np.ones([2, 2]),
```

Comment:
yeah, we definitely need to fix this behavior in the next PR. it's very confusing.

Reply:
I need to look at the BatchInputParser class a bit before we can merge